assets/queries/openAPI/path_server_uses_http/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+import data.generic.openapi as openAPILib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openAPILib.checkOpenAPI(doc) != "undefined"
+	server := doc.servers[n]
+
+	regex.match("^(http:)", server.url)
+
+	result := {
+		"documentId": doc.id,
+		"searchKey": sprintf("openapi.servers.url={{%s}}", [server.url]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": "A default security schema should be defined",
```

Comment:
Shouldn't this be something like: "Server URL should use 'https' protocol instead of 'http'"?